build.ts

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env
+import { variants } from "npm:@catppuccin/palette@0.1.5";
+import { titleCase } from "https://deno.land/x/case@2.2.0/mod.ts";
+import { JSZip } from "https://deno.land/x/jszip@0.11.0/mod.ts";
+
+const accents = [
+  "rosewater",
+  "flamingo",
+  "pink",
+  "mauve",
+  "red",
+  "maroon",
+  "peach",
+  "yellow",
+  "green",
+  "teal",
+  "sky",
+  "sapphire",
+  "blue",
+  "lavender",
+] as const;
+
+const start = performance.now();
+let count = 0;
+
+await Promise.all(
+  Object.entries(variants).map(async ([name, palette]) => {
+    for (const accent of accents) {
+      const newTheme = {
+        manifest_version: 2,
+        name: `catppuccin-${name}-${accent}`,
+        version: "1.0.0",
+        applications: {
+          gecko: {
+            // FIXME: Make properly
+            id: `theme@catppuccinorg.com`,
+            strict_min_version: "60.0",
+          },
+        },
+        description: `Catppuccin theme for Thunderbird - ${titleCase(
```

Comment:
```suggestion
        description: `Soothing pastel theme for Thunderbird - ${titleCase(
```
Not sure if that sort of description format would work here? :p